docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: PATCH updating storage
+---
+
+
+`PATCH /asset/:id -d {"storage": {"ipfs": {}}` can be used to modify any `asset` that was stored with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
+
+
+
+## Update Stored Asset
+
+> **NOTE:** When updating the asset information, the `storage` object must be included in the body of the request
```

Comment:
We should updated the API to make this not necessary. Users should be able to update each property independently.